src/main/java/technology/semi/weaviate/client/Config.java

```diff
@@ -13,6 +18,29 @@ public Config(String scheme, String host, Map<String, String> headers) {
     this.host = host;
     this.version = "v1";
     this.headers = headers;
+    this.connectionTimeoutMs = DEFAULT_TIMEOUT;
```

Comment:
can be expressed as call to other constructor:
`this(scheme, host, headers, DEFAULT_TIMEOUT, DEFAULT_TIMEOUT, DEFAULT_TIMEOUT);`
or
`this(scheme, host, headers, DEFAULT_TIMEOUT);`

but that is also ok